src/views/Permissions.vue

```diff
@@ -0,0 +1,296 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-title>{{ translate("Users management") }}</ion-title>
+        <ion-buttons slot="end">
+          <ion-button @click="downloadCSVForPermissions()">
+            <ion-icon :icon="downloadOutline" slot="icon-only" />
+          </ion-button>
+        </ion-buttons>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content>
+      <div class="find">
+        <aside>
+          <h1>{{ translate("Security Groups") }}</h1>
+
+          <ion-list>
+            <ion-item v-for="group in securityGroups" :key="group?.groupId" button detail @click="updateCurrentGroup(group)">
+              <ion-label :color="group?.groupId === currentGroup?.groupId ? 'primary' : ''">
+                <p class="overline">{{ group?.groupId }}</p>
+                {{ group?.groupName }}
+              </ion-label>
+            </ion-item>
+          </ion-list>
+
+          <ion-button @click="createGroup()" fill="clear" expand="block">
+            <ion-icon slot="start" :icon="addOutline" />
+            <ion-label>{{ translate("Create security group") }}</ion-label>
+          </ion-button>
+        </aside>
+        <main v-if="Object.keys(currentGroup).length">
```

Comment:
Instead of this, we can use currentGroup.groupId, considering that currentGroup is always an object.